bin-src/lib/getStorybookInfo.test.ts

```diff
@@ -100,19 +140,63 @@ describe('getStorybookInfo', () => {
         addons: [],
         version: null,
         viewLayer: null,
+        builder: null,
       });
     });
   });
 
   describe('with --storybook-build-dir', () => {
+    beforeAll(() => {
+      mock({
+        'storybook-static': {
+          'project.json':
+            '{"generatedAt":1654054690474,"builder":{"name":"webpack5"},"hasCustomBabel":false,"hasCustomWebpack":true,"hasStaticDirs":false,"hasStorybookEslint":false,"refCount":0,"packageManager":{"type":"yarn","version":"1.22.18"},"features":{"postcss":false},"storybookVersion":"6.5.6","language":"typescript","storybookPackages":{"@storybook/addon-essentials":{"version":"6.5.6"},"@storybook/builder-webpack5":{"version":"6.5.6"},"@storybook/eslint-config-storybook":{"version":"3.1.2"},"@storybook/linter-config":{"version":"3.1.2"},"@storybook/manager-webpack5":{"version":"6.5.6"},"@storybook/react":{"version":"6.5.6"}},"framework":{"name":"react"},"addons":{"@storybook/addon-viewport":{"version":"6.5.6"}}}',
+        },
+      });
```

Comment:
Is this mock even used, considering you're mocking out `./getPrebuiltStorybookMetadata` at the top? I don't see you disabling that mock to call through to the actual module. If it is, maybe you can change some mock data and the assertions along with it so that they don't overlap with the other mock (e.g. use webpack5 v6.5.4 for these tests, rather than 6.5.6).

Reply:
Resolved this by pointing at a mock project.json file that allows that actually uses the functions instead of mocking.